test/development/app-dir/capture-console-error-owner-stack/capture-console-error-owner-stack.test.ts

```diff
@@ -220,7 +220,12 @@ describe('app-dir - capture-console-error-owner-stack', () => {
          3 |   return <p>rsc</p>
          4 | }
          5 |",
-         "title": "Console Error",
+         ${
+           process.env.__NEXT_EXPERIMENTAL_NEW_DEV_OVERLAY === 'true'
+             ? `"title": "Console Error
```

Comment:
We should snapshot the label as another property, this won't tell if "Server" is part of the message or it's a separate component